docs/source/science/ESA_CCI/ESA_CCI_V4.ipynb

```diff
@@ -7,7 +7,7 @@
     "tags": []
```

Comment:
<p>I'd remove the first-line comment to avoid users from uncommenting the entire cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>